render.js

```diff
@@ -0,0 +1,50 @@
+// render :: (string, string) -> Node
+function render(domId, content) {
+  const el = document.getElementById(domId).innerHTML = content;
+  return el;
+}
+
+function renderPeopleToEatWith(domId, listOfPeople) {
+  const formId = 'people-to-select';
+  const selectName = 'select-people';
+  const peopleInOption = listOfPeople.map(cur => `<option value="${cur.id}">${cur.name}</option>`);
+  const select = `<form id="${formId}">
```

Comment:
and what is this? it this JSX? or String interpolations?

Reply:
String interpolations, JSX is old school...